tests/nightly/unity/task.yaml

```diff
@@ -28,4 +32,6 @@ execute: |
 
     echo "Then the app window is created"
     expected=".*?\"qmlscene: clockMainView\": \(\"qmlscene\" \"com\.ubuntu\.clock\"\)"
-    while ! xwininfo -tree -root | grep -Pq "$expected"; do sleep 1; done
+    while ! xwininfo -tree -root | grep -Pq "$expected";
+        do sleep 1;
```

Comment:
The location of `do` is unusual here. Could you please move it to the previous line?